### PR TITLE
Updated the libcontainer interface comments

### PR DESCRIPTION
**Pull request by rajasec**
Updated the libcontainer interface comments as per error code

Signed-off-by: rajasec <rajasec79@gmail.com>

### CODE REVIEW

File: libcontainer/container.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ type BaseContainer interface {
 	// Returns the current status of the container.
 	//
 	// errors:
-	// ContainerDestroyed - Container no longer exists,
+	// ContainerNotExists - Container no longer exists,
```
Comment (hqhq) [inline libcontainer/container.go]: I think they are not returning `ContainerNotExists`.

**Comment by crosbymichael** [comment]
Can you rebase this change?  Thanks!

**Comment by rajasec** [comment]
Rebased.

**Comment by crosbymichael** [comment]
LGTM

**Comment by hqhq** [comment]
LGTM

**Pull request merged**